pkg/asset/machines/master.go

```diff
@@ -250,17 +250,19 @@ func Machines(files []*asset.File) ([]machineapi.Machine, error) {
 	return machines, nil
 }
 
-// IsMasterManifest tests whether a file is a manifest that belongs to the
-// Master Machines asset.
-func IsMasterManifest(file *asset.File) bool {
+// IsMachineManifest tests whether a file is a manifest that belongs to the
+// Master Machines or Worker Machines asset.
+func IsMachineManifest(file *asset.File) bool {
 	if filepath.Dir(file.Filename) != directory {
 		return false
 	}
 	filename := filepath.Base(file.Filename)
-	if filename == masterUserDataFileName {
+	if filename == masterUserDataFileName || filename == workerUserDataFileName {
 		return true
 	}
-	if machineconfig.IsManifest("master", filename) {
+	if matched, err := machineconfig.IsManifest(filename); err != nil {
+		panic("bad format for MachineConfig file name pattern")
```

Comment:
how does the caller know that error was because of bad format?

Reply:
> how does the caller know that error was because of bad format?

Because I wrote them both ;).  Changed to just `panic(err)` as part of c0bc4dc -> 009963e.